src/js/rx/components/ErrorMessages.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const ErrorMessages = ({ errors }) => {
+  if (errors && errors.length > 0) {
+    return (<span>
+      <br/>
+      {
+        errors.map(e => {
+          return (
+            <span key={e.code}>
+              Error {e.code}: {e.detail}
+              <br/>
```

Comment:
Even these `span` elements can be converted to block-level elements without the `br` if there's going to be a line break between each one. Maybe the outer can be a `div` and the inner errors can be `p` elements. Or `ul` and `li` elements, which might be more semantic.